node/node_syncing.go

```diff
@@ -477,3 +470,49 @@ func (node *Node) CalculateResponse(request *downloader_pb.DownloaderRequest, in
 	}
 	return response, nil
 }
+
+const (
+	headerCacheSize = 50000000
```

Comment:
This number is not not size in byte. It's number of items.

Reply:
Fixed. Thanks!

Reply:
Should be 10000

Reply:
Should be 10000